sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -94,6 +67,40 @@ public static void Verify(Compilation compilation, DiagnosticAnalyzer diagnostic
             }
         }
 
+        private static void CompareActualToExpected(IEnumerable<Diagnostic> diagnostics, List<IIssueLocation> expectedIssues, bool compareIdToMessage)
+        {
+            foreach (var diagnostic in diagnostics)
+            {
+                VerifyIssue(expectedIssues,
+                    issue => issue.IsPrimary,
+                    diagnostic.Location,
+                    compareIdToMessage ? diagnostic.Id : diagnostic.GetMessage(),
+                    compareIdToMessage ? diagnostic.Id + ": " + diagnostic.GetMessage() : null,
```

Comment:
The web-based diff in GitHub makes it difficult to see if what if anything has changed in when blocks of code are moved to a new function. It really helps reviewers if you call in out a review comment what has changed (or that nothing has changed and the code has just been moved).